scripts/generate_proto_stubs.sh

```diff
@@ -45,7 +45,7 @@ PYTHON_PROTOBUF_DIR="protobuf-$PYTHON_PROTOBUF_VERSION"
 VENV=venv
 python3 -m venv "$VENV"
 source "$VENV/bin/activate"
-pip install -r "$REPO_ROOT/requirements-tests.txt"  # for Black and Ruff
+pip install pre-commit
 
 # Install mypy-protobuf
 pip install mypy-protobuf=="$MYPY_PROTOBUF_VERSION"
```

Comment:
If you wanna be more consistant between scripts
```suggestion
pip install pre-commit mypy-protobuf=="$MYPY_PROTOBUF_VERSION"
```